### ISSUE TITLE
Workflows DIR not exist w/ new Alfred install

### ISSUE THREAD

**Issue by F1LT3R** [sindresorhus/alfred-npms#21]
```shell
 npm install --global alfred-npms

> alfred-npms@1.1.0 postinstall /usr/local/lib/node_modules/alfred-npms
> alfy-init

{ Error: Command failed: alfred-link
Error: Workflow directory `/Users/dubba/repos/dotfiles-macbook-pro/alfred/Alfred.alfredpreferences/workflows` does not exist
    at pathExists.then.exists (/usr/local/lib/node_modules/alfred-npms/node_modules/alfred-link/index.js:19:10)
    at <anonymous>

    at Promise.all.then.arr (/usr/local/lib/node_modules/alfred-npms/node_modules/execa/index.js:201:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
  code: 1,
  killed: false,
  stdout: '',
  stderr: 'Error: Workflow directory `/Users/dubba/repos/dotfiles-macbook-pro/alfred/Alfred.alfredpreferences/workflows` does not exist\n    at pathExists.then.exists (/usr/local/lib/node_modules/alfred-npms/node_modules/alfred-link/index.js:19:10)\n    at <anonymous>\n',
  failed: true,
  signal: null,
  cmd: 'alfred-link',
  timedOut: false }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! alfred-npms@1.1.0 postinstall: `alfy-init`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the alfred-npms@1.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dubba/.npm/_logs/2018-04-12T19_01_01_940Z-debug.log
```

**Comment by F1LT3R** [comment]
FYI: Works fine if I mkdir the `workflows` directory manually.

**Comment by sindresorhus** [comment]
This should be fixed in v2.0.0. 

Original issue: https://github.com/sindresorhus/alfy/issues/80
